week_3/week_3.txt

```diff
@@ -14,4 +14,16 @@
         - innerText
         - querySelector
         - querySelectorAll
-        - classList
\ No newline at end of file
+        - classList
+
+16 APRIL
+    Create a Todo for practice
+    Create rock paper scissors for practice
+    addEventListeners
+    setInterval
+    setTimeout
+    Starting an amazon project for practice
+    Modules 
+    Data Attributes
+    local storage
```

Comment:
@codal-kcolaco please go-thorough with shadow copy and shwllow copy in js and how we can solve it?